### PR TITLE
*: add a limitation about not supporting `NULLIF()` and supporting `CASE`

### PR DESCRIPTION
**Pull request by Frank945946**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: generated-columns.md [hunk 1]
```diff
@@ -159,4 +159,5 @@ desc select a+1 from t where a+1=3;
 - 不能通过 `ALTER TABLE` 将存储生成列转换为普通列，也不能将普通列转换成存储生成列；
 - 不能通过 `ALTER TABLE` 修改存储生成列的生成列表达式；
 - 并未支持所有的 [JSON 函数](/functions-and-operators/json-functions.md)；
+- 不支持使用 [NULLIF() 函数](/control-flow-functions#nullif.md)，可使用 [Case 函数](/control-flow-functions#case.md) 代替。
```
Comment (hfxsd) [inline generated-columns.md:162]:
```suggestion
- 不支持使用 [`NULLIF()` 函数](/control-flow-functions.md#nullif)。可以使用 [`CASE` 函数](/control-flow-functions.md#case)代替。
```

**Review by hfxsd — APPROVED** [review]

File: generated-columns.md [hunk 2]
```diff
@@ -159,4 +159,5 @@ desc select a+1 from t where a+1=3;
 - 不能通过 `ALTER TABLE` 将存储生成列转换为普通列，也不能将普通列转换成存储生成列；
 - 不能通过 `ALTER TABLE` 修改存储生成列的生成列表达式；
 - 并未支持所有的 [JSON 函数](/functions-and-operators/json-functions.md)；
+- 不支持使用 [`NULLIF()` 函数](/control-flow-functions.md#nullif)。可以使用 [`CASE` 函数](/control-flow-functions.md#case)代替。
```
Comment (hfxsd) [inline generated-columns.md:162]:
```suggestion
- 不支持使用 [`NULLIF()` 函数]((/functions-and-operators/control-flow-functions.md#nullif)。可以使用 [`CASE` 函数]((/functions-and-operators/control-flow-functions.md#case)代替。
```

File: generated-columns.md [hunk 3]
```diff
@@ -159,4 +159,5 @@ desc select a+1 from t where a+1=3;
 - 不能通过 `ALTER TABLE` 将存储生成列转换为普通列，也不能将普通列转换成存储生成列；
 - 不能通过 `ALTER TABLE` 修改存储生成列的生成列表达式；
 - 并未支持所有的 [JSON 函数](/functions-and-operators/json-functions.md)；
+- 不支持使用 [`NULLIF()` 函数]((/functions-and-operators/control-flow-functions.md#nullif)。可以使用 [`CASE` 函数]((/functions-and-operators/control-flow-functions.md#case)代替。
```
Comment (hfxsd) [inline generated-columns.md:162]:
```suggestion
- 不支持使用 [`NULLIF()` 函数](/functions-and-operators/control-flow-functions.md#nullif)。可以使用 [`CASE` 函数]((/functions-and-operators/control-flow-functions.md#case)代替。
```

File: generated-columns.md [hunk 4]
```diff
@@ -159,4 +159,5 @@ desc select a+1 from t where a+1=3;
 - 不能通过 `ALTER TABLE` 将存储生成列转换为普通列，也不能将普通列转换成存储生成列；
 - 不能通过 `ALTER TABLE` 修改存储生成列的生成列表达式；
 - 并未支持所有的 [JSON 函数](/functions-and-operators/json-functions.md)；
+- 不支持使用 [`NULLIF()` 函数](/functions-and-operators/control-flow-functions.md#nullif)。可以使用 [`CASE` 函数]((/functions-and-operators/control-flow-functions.md#case)代替。
```
Comment (lilin90) [inline generated-columns.md:162]:
```suggestion
- 不支持使用 [`NULLIF()` 函数](/functions-and-operators/control-flow-functions.md#nullif)；可以使用 [`CASE` 函数](/functions-and-operators/control-flow-functions.md#case)代替。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**